packages/wrangler/src/hyperdrive/update.ts

```diff
@@ -94,23 +106,12 @@ export async function handler(
 	args: StrictYargsOptionsToInterface<typeof options>
 ) {
 	// check if all or none of the required origin fields are set, since we don't allow partial updates of the origin
-	const allOriginFieldsSet = requiredOriginOptions.every((field) =>
+	const coreOriginFieldsSet = coreOriginOptions.every((field) =>
 		isOptionSet(args, field)
 	);
-	const noOriginFieldSet = requiredOriginOptions.every(
-		(field) => !isOptionSet(args, field)
-	);
-
-	if (!allOriginFieldsSet && !noOriginFieldSet) {
-		throw new UserError(
-			`When updating the origin, all of the following must be set: ${requiredOriginOptions
-				.map((option) => camelToKebab(option))
-				.join(", ")}`
-		);
```

Comment:
Why is this being removed? It's a much more user friendly error than the default yargs one:

```
Implications failed:
   database -> origin-host database -> origin-user database -> origin-password
```

Reply:
I prefer to let yargs handle that instead of writing my own logic and risk introducing bugs.